dbms/src/Storages/MergeTree/MergeTreeData.cpp

```diff
@@ -1030,8 +1030,11 @@ MergeTreeData::AlterDataPartTransactionPtr MergeTreeData::alterDataPart(
             *this, part, DEFAULT_MERGE_BLOCK_SIZE, 0, 0, expression->getRequiredColumns(), ranges,
             false, nullptr, "", false, 0, DBMS_DEFAULT_BUFFER_SIZE, false);
 
+        auto compression_method = this->context.chooseCompressionMethod(
+            this->getTotalActiveSizeInBytes(),
```

Comment:
Does it mean I should take   this->getTotalActiveSizeInBytes() and divide it by total compressed size? I'm asking because I'm not fully aware of what methods stands for what values and I am not much familiar with CH code.